src/test/java/seedu/address/logic/commands/ListAppointmentCommandTest.java

```diff
@@ -0,0 +1,5 @@
+package seedu.address.logic.commands;
+
+public class ListAppointmentCommandTest {
+
+}
```

Comment:
Currently this is just a blank test. I assume we'll work on this next iteration, but I'll mark this as a comment so everyone in this conversation is aware.